x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/epm/screens/home/index.tsx

```diff
@@ -67,29 +67,42 @@ export function EPMHomePage() {
 function InstalledPackages() {
   const { data: allPackages, isLoading: isLoadingPackages } = useGetPackages();
   const [selectedCategory, setSelectedCategory] = useState('');
-  const packages =
-    allPackages && allPackages.response && selectedCategory === ''
-      ? allPackages.response.filter(pkg => pkg.status === 'installed')
-      : [];
 
   const title = i18n.translate('xpack.ingestManager.epmList.installedTitle', {
     defaultMessage: 'Installed integrations',
   });
 
+  const allInstalledPackages =
+    allPackages && allPackages.response
+      ? allPackages.response.filter(pkg => pkg.status === 'installed')
+      : [];
+
+  const updatablePackages = allInstalledPackages.reduce<PackageListItem[]>((acc, item) => {
+    if ('savedObject' in item) {
+      if (item.version > item.savedObject.attributes.version) {
+        return acc.concat(item);
+      }
+    }
+    return acc;
+  }, []);
+
+  const packages =
+    selectedCategory === 'updates_available' ? [...updatablePackages] : [...allInstalledPackages];
```

Comment:
Why copy these? Does returning *Packages cause issues?

Reply:
I guess I just try to never refer to arrays to avoid possible surprises.